### PR TITLE
Enable additional sidecar containers

### PR DESCRIPTION
**Pull request by ArchiFleKs**
Enable additional containers as sidecar.

### CODE REVIEW

File: chart/flux/templates/deployment.yaml [hunk 1]
```diff
@@ -218,14 +218,17 @@ spec:
 {{ toYaml .Values.additionalArgs | indent 10 }}
           {{- end }}
           resources:
+{{ toYaml .Values.resources | indent 12 }}
+{{- if .Values.controller.extraContainers }}
```
Comment (hiddeco) [inline chart/flux/templates/deployment.yaml]:
```suggestion
{{- if .Values.extraContainers }}
```

Comment (ArchiFleKs) [reply chart/flux/templates/deployment.yaml]: done

Comment (hiddeco) [reply chart/flux/templates/deployment.yaml]: You fixed the line below but missed the one in the `if` statement :-)

**Comment by hiddeco** [comment]
@ArchiFleKs thanks for the fixes, I am going to quickly rebase this to get rid of the 'merge branch' commits to keep our `master` history clean.

**Comment by ArchiFleKs** [comment]
@hiddeco should we update the charts minor version or is it going to be automatically available on https://weaveworks.github.io/flux as 0.9.5 ?

**Comment by hiddeco** [comment]
@ArchiFleKs a new release will be put out tomorrow after we have released Flux `1.13.0`.

**Pull request merged**